app/products/calls/hooks.ts

```diff
@@ -108,3 +109,27 @@ export const usePermissionsChecker = (micPermissionsGranted: boolean) => {
         }
     }, [appState]);
 };
+
+export const useCallsAdjustment = (serverUrl: string, channelId: string) => {
```

Comment:
Isn't this from the other PR?

Reply:
Yes, I'm not sure why it's showing a diff here, it's on main. 🤷